src/schema/search.ts

```diff
@@ -388,6 +410,37 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         .limit(getSearchLimit({ limit }));
       const hits = await searchQuery.getRawMany();
 
+      return {
+        query,
+        hits,
+      };
+    },
+    searchUserSuggestions: async (
+      source,
+      { query, limit }: SearchSuggestionArgs,
+      ctx,
+    ): Promise<GQLSearchSuggestionsResults> => {
+      const searchQuery = ctx.con
+        .createQueryBuilder()
+        .select(`id, name as title, username as subtitle, image`)
+        .from('user', 'u')
+        .where('u.infoConfirmed = TRUE')
+        .andWhere("u.id != '404'")
```

Comment:
Instead of hardcoding `404`, lets use the `ghostUser.id` variable?

Reply:
Good point, probably should fix it in some other places as well!